ValheimServer/ValheimServer/MainWindow.xaml.cs

```diff
@@ -24,5 +25,11 @@ public MainWindow()
         {
             InitializeComponent();
         }
+
+        private void SteamcmdButton_Click(object sender, RoutedEventArgs e)
+        {
+            //MessageBox.Show("The button works");
+            Process.Start("steamcmd.exe");
```

Comment:
Nick i saw your comment about being able to close the window after its done. 
It is doable bc steamcmd has an argument to close itsself.
On command line it would like like this: steamcmd.exe +quit

The Process.Start() method has a way to pass arguments.
`Process.Start("steamcmd.exe", "+quit");`